### PR TITLE
Introducing `color()` `oklab/oklch`, `ic` to PostCSS Preset Env

### PR DESCRIPTION
**Pull request by Antonio-Laguna**
* Integrates both in PostCSS Preset Env and CLI the following:
  * `@csstools/color-function`
  * `@csstools/oklab-function`
  * `@csstools/ic-unit`
* Also fixes README and add missing options to CLI plugins as well as some logger texts

### CODE REVIEW

File: plugin-packs/postcss-preset-env/test/insert.after.match-result.expect.css [hunk 1]
```diff
@@ -2,5 +2,5 @@
 	background-color: rgba(255, 255, 255, 0);
 	background-color: color(display-p3 1.00000 1.00000 1.00000 / 0%);
 	after: changed-this-declaration;
-	color: color(display-p3 0.00000 0.00000 0.00000 / 0%);
+	after: changed-this-declaration;
```
Comment (Antonio-Laguna) [inline plugin-packs/postcss-preset-env/test/insert.after.match-result.expect.css:5]: Something odd going on here, it gets retransformed somehow. These three tests on this commit seem wrong

File: plugin-packs/postcss-preset-env/.tape.mjs [hunk 2]
```diff
@@ -349,6 +329,38 @@ postcssTape(plugin)({
...
+			],
+			insertAfter: {
+				'lab-function': [
+					orderDetectionPlugin('after', (decl) => {
+						return decl.prop === 'color' && decl.value !== 'changed-this-declaration';
+					})
+				]
+			}
+		}
+	},
```
Comment (romainmenke) [inline plugin-packs/postcss-preset-env/.tape.mjs:363]: This two test cases had copy/paste issues.
Was intended to test `insertBefore` and `insertAfter` when the attached feature is skipped.

- messages was not acurate
- both had the same config

I also placed them lower in the list.

File: plugin-packs/postcss-preset-env/.tape.mjs [hunk 3]
```diff
@@ -271,44 +280,13 @@ postcssTape(plugin)({
...
-		}
-	},
 	'insert:after:match-source': {
 		message: 'supports { insertAfter } usage when looking for source',
 		options: {
 			stage: 0,
 			features: {
-				'lab-function': true
+				'lab-function': true,
+				'color-function': false,
```
Comment (romainmenke) [inline plugin-packs/postcss-preset-env/.tape.mjs:289]: `color-function` transforms `color(display-p3 ...)`.
These tests look for the results of `lab-function` transforms, so displaying `color-function` for now in these cases.

Maybe these need to be rewritten to use a more basic plugin?

Comment (Antonio-Laguna) [reply plugin-packs/postcss-preset-env/.tape.mjs:289]: I thought it was related to this and also thought about using something else such as `rebeccapurple` or something that's less prone to change.

Comment (Antonio-Laguna) [reply plugin-packs/postcss-preset-env/.tape.mjs:289]: Thanks for taking care of this!

**Review by romainmenke — APPROVED** [review]
LGTM!

Thank you for carefully adding all the new things to both `preset-env` and the `cli`! 🎉

**Pull request merged**